src/sssom/util.py

```diff
@@ -105,6 +105,59 @@ def with_converter(
             metadata=metadata or get_default_metadata(),
         )
 
+    @classmethod
+    def from_mappings(
+        cls,
+        mappings: List[SSSOM_Mapping],
+        *,
+        converter: HINT = None,
+        metadata: Optional[MetadataType] = None,
+    ) -> "MappingSetDataFrame":
```

Comment:
Is this a trick so you can specify the return type (for use in code quality checking tools) without importing it?

Reply:
yes, since the name of the class isn't avaliable in the global namespace until after it's parsed. You can also do `from __future__ import annotations` to get around this too